app/src/main/java/io/branch/branchlinksimulator/ApiSettingsPanel.kt

```diff
@@ -0,0 +1,162 @@
+        ApiInfoRow(label = "Branch Key", value = selectedConfig.branchKey)
+        ApiInfoRow(label = "API URL", value = selectedConfig.apiUrl)
+        ApiInfoRow(label = "App ID", value = selectedConfig.appId)
+
+        Column(horizontalAlignment = Alignment.CenterHorizontally) {
+            Row(horizontalArrangement = Arrangement.spacedBy(5.dp)) {
+                ApiButton(configName = STAGING, selectedConfig = selectedConfig, Modifier.weight(1f), onSelect = {
+                    selectedConfig = it
+                    saveConfig(preferences, it)
+                })
+                ApiButton(configName = PRODUCTION, selectedConfig = selectedConfig, Modifier.weight(1f), onSelect = {
+                    selectedConfig = it
+                    saveConfig(preferences, it)
+                })
+            }
+
+            Row(horizontalArrangement = Arrangement.spacedBy(5.dp)) {
+                ApiButton(configName = STAGING_AC, selectedConfig = selectedConfig, Modifier.weight(1f), onSelect = {
+                    selectedConfig = it
+                    saveConfig(preferences, it)
+                })
+                ApiButton(configName = PRODUCTION_AC, selectedConfig = selectedConfig, Modifier.weight(1f), onSelect = {
+                    selectedConfig = it
+                    saveConfig(preferences, it)
+                })
+            }
+        }
+    }
+}
+
+@Composable
+fun ApiInfoRow(label: String, value: String) {
+    val localClipboardManager = LocalClipboardManager.current
+
+    Row(
+        modifier = Modifier
+            .background(Color.LightGray, RoundedCornerShape(10.dp))
+            .padding(horizontal = 10.dp, vertical = 5.dp),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        Column(
+            modifier = Modifier.weight(1f)
+        ) {
+            Text(
+                text = "$label:",
+                style = MaterialTheme.typography.labelMedium,
+                color = Color.Black
+            )
+            Text(
+                text = value,
+                style = MaterialTheme.typography.bodySmall,
+                color = Color.Black
+            )
+        }
+
+        Button(
+            onClick = { copyToClipboard(localClipboardManager, value) },
+        ) {
+            Text("Copy")
+        }
+    }
+}
+
+
+@Composable
+fun ApiButton(
+    configName: String,
+    selectedConfig: ApiConfiguration,
+    modifier: Modifier = Modifier,
+    onSelect: (ApiConfiguration) -> Unit
+) {
+    val config = apiConfigurationsMap[configName]
+    val isSelected = selectedConfig == config
+
+    var showDialog by remember { mutableStateOf(false) }
+
+    Button(
+        onClick = {
+            if (config != null) {
+                onSelect(config)
+                showDialog = true
+            }
+        },
+        modifier = modifier.padding(0.dp),
+        colors = ButtonDefaults.buttonColors(
+            containerColor = if (isSelected) Color.Blue else Color.Gray,
```

Comment:
This themed blue matches the color of the other buttons. 
```suggestion
            containerColor = if (isSelected) MaterialTheme.colorScheme.primary else Color.Gray,
```